src/jira.js

```diff
@@ -705,6 +705,41 @@ export default class JiraApi {
     );
   }
 
+  /** Returns a user.
+   * [Jira Doc](https://developer.atlassian.com/cloud/jira/platform/rest/v3/#api-rest-api-3-user-get)
+   * @name getUser
+   * @function
+   * @param {string} accountId - The accountId of user to search for
+   * @param {string} expand - The expand for additional info (groups,applicationRoles)
+   */
+  getUser(accountId, expand) {
+    const query = {
+      accountId,
+      expand,
+    };
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: '/user',
+      query,
```

Comment:
Nitpicking: Can you put directly the parameters here without declaring `const query`? For the sake of consistency with `getUsers` that you did below.
```js
{
  pathname: '/user',
  query: {
    accountId,
    expand
  }
}
```

Reply:
Sure, i have fixed it

Reply:
Awesome, thanks a lot!